integration/shareddata/composites.go

```diff
@@ -66,13 +66,27 @@ var DocumentsDoubles = &Values[string]{
 	handlers: []string{"pg", "tigris"},
 	validators: map[string]map[string]any{
 		"tigris": {
-			"$tigrisSchemaString": tigrisSchema(`"type": "object", "properties": {"v": {"type": "number"}}`),
+			"$tigrisSchemaString": `{
+				"title": "%%collection%%",
+				"primary_key": ["_id"],
+				"properties": {
+					"v": {
+						"type": "object", 
+						"properties": {
+							"foo": {"type": "object", "properties": {"bar": {"type": "string"}} }, 
+							"v": {"type": "number"}
+						}
+					},
+					"_id": {"type": "string"}
+				}
+			}`,
 		},
 	},
 	data: map[string]any{
 		"document-double":          bson.D{{"v", 42.13}},
 		"document-double-whole":    bson.D{{"v", 42.0}},
 		"document-double-zero":     bson.D{{"v", 0.0}},
+		"document-double-unset":    bson.D{{"foo", nil}},
```

Comment:
Let's not confuse null values with unset values:

https://github.com/FerretDB/FerretDB/blob/b348a249df5bea4282dbb3fde03965150882c7d3/integration/shareddata/shareddata.go#L25-L26
https://github.com/FerretDB/FerretDB/blob/b348a249df5bea4282dbb3fde03965150882c7d3/integration/shareddata/shareddata.go#L184-L186

Reply:
renamed